scripts/utils/test.js

```diff
@@ -1,46 +1,106 @@
 const hre = require("hardhat");
```

Comment:
Generally, the file must be marked what its for, so either mark it, please via comment, or remove.